x/auctions/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,153 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package keeper_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	testutil "github.com/evmos/evmos/v19/testutil"
+	testnetwork "github.com/evmos/evmos/v19/testutil/integration/evmos/network"
+	testutiltx "github.com/evmos/evmos/v19/testutil/tx"
+	"github.com/evmos/evmos/v19/utils"
+	"github.com/evmos/evmos/v19/x/auctions/types"
+)
+
+func TestAuctionInfo(t *testing.T) {
+	var network *testnetwork.UnitTestNetwork
+	ibcCoin := sdk.NewCoin("ibc/D219F3A490310B65BDC312B5A644B0D56FFF1789D894B902A49FBF9D2F560B32", sdk.NewInt(1))
+	evmosCoin := sdk.NewCoin("aevmos", sdk.NewInt(1))
+	anotherNativeCoin := sdk.NewCoin("aeth", sdk.NewInt(1))
+	validBidderAddr, _ := testutiltx.NewAccAddressAndKey()
+
+	testCases := []struct {
+		name        string
+		malleate    func()
+		expResp     func() *types.QueryCurrentAuctionInfoResponse
+		expPass     bool
+		errContains string
+	}{
+		{
+			name: "pass with default genesis state",
+			malleate: func() {
+			},
+			expResp: func() *types.QueryCurrentAuctionInfoResponse {
+				return &types.QueryCurrentAuctionInfoResponse{
+					Tokens:        nil,
+					CurrentRound:  0,
+					HighestBid:    sdk.NewCoin(utils.BaseDenom, sdk.NewInt(0)),
+					BidderAddress: "",
+				}
+			},
+			expPass:     true,
+			errContains: "",
```

Comment:
nit: the non-filled entries could be ditched for clearer testcases (IMO, debatable I guess)


```suggestion
			name: "pass with default genesis state",
			expResp: func() *types.QueryCurrentAuctionInfoResponse {
				return &types.QueryCurrentAuctionInfoResponse{
					Tokens:        nil,
					CurrentRound:  0,
					HighestBid:    sdk.NewCoin(utils.BaseDenom, sdk.NewInt(0)),
					BidderAddress: "",
				}
			},
			expPass:     true,
```

Reply:
I usually prefer to not hide anything used in tests, but probably the one you pointed out the approach usually followed in our tests 🤔 will go for your suggestion